src/ai/backend/common/web/session/__init__.py

```diff
@@ -11,7 +11,7 @@
 
 from ai.backend.common.logging import BraceStyleAdapter
 
-log = BraceStyleAdapter(logging.getLogger("ai.backend.web.server"))
+log = BraceStyleAdapter(logging.getLogger("ai.backend.manager.scheduler"))
```

Comment:
Here seems to have a wrong name for logger.

```suggestion
log = BraceStyleAdapter(logging.getLogger("ai.backend.common.web.session"))
```